lib/core.ps1

```diff
@@ -307,6 +333,7 @@ function Get-HelperPath {
     }
     process {
         switch ($Helper) {
+            'Git' { $HelperPath = "$(versiondir 'git' 'current')\mingw64\bin\git.exe" }
```

Comment:
Well this is a problem because I don't use Git installed by Scoop 😅

Ideally there should be a fallback here, which can `Test-Path` this path and fall back to `Get-Command`.

Reply:
Dang it, I knew it. :grin:

Yeah, will implement a fallback :smile: